File: 백준/Silver/10814. 나이순 정렬/나이순 정렬.py
import sys
input = sys.stdin.readline

case = int(input())
user_info = []

for _ in range(case):
    user = list(input().split())
    user[0] = int(user[0])
    user_info.append(user)

user_info.sort(key=lambda x : x[0])

for i in user_info:
    print(i[0], i[1])

File: 백준/Silver/10814. 나이순 정렬/나이순 정렬.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const [, ...targetInput] = input;

const result = targetInput.map(item => item.split(" ")).sort((a, b) => Number(a[0]) - Number(b[0]))

console.log(result.map(item => item.join(" ")).join("\n"))